docs/modeling/relationships.rst

```diff
@@ -0,0 +1,236 @@
+ - **Reference navigation property:** A navigation property that holds a reference to a single related entity.
+ - **Inverse navigation property:** When discussing a particular navigation property, this term refers to the navigation property on the other end of the relationship.
+
+The following code listing shows a one-to-many relationship between ``Blog`` and ``Post``
+  - ``Post`` is the dependent entity
+  - ``Blog`` is the principal entity
+  - ``Post.BlogId`` is the foreign key
+  - ``Blog.BlogId`` is the principal key (in this case it is a primary key rather than an alternate key)
+  - ``Post.Blog`` is a reference navigation property
+  - ``Blog.Posts`` is a collection navigation property
+  - ``Post.Blog`` is the inverse navigation property of ``Blog.Posts`` (and vice versa)
+
+.. literalinclude:: configuring/sample/EFModeling.Conventions/Samples/RelationshipFull.cs
+        :language: c#
+        :lines: 12-28
+        :linenos:
+
+Conventions
+-----------
+
+By convention, a relationship will be created when there is a navigation property discovered on a type. A property is considered a navigation property if the type it points to can not be mapped as a scalar type by the current database provider.
+
+.. note::
+    Relationships that are discovered by convention will always target the primary key of the principal entity. To target an alternate key, additional configuration must be performed using the Fluent API.
+
+Fully Defined Relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The most common pattern for relationships is to have navigation properties defined on both ends of the relationship and a foreign key property defined in dependent entity class.
+ - If a pair of navigation properties is found between two types, then they will be configured as inverse navigation properties of the same relationship.
+ - If the dependent entity contains a property named ``<primary key property name>``, ``<navigation property name><primary key property name>``, or ``<principal entity name><primary key property name>`` then it will be configured as the foreign key.
+
+.. literalinclude:: configuring/sample/EFModeling.Conventions/Samples/RelationshipFull.cs
+       :language: c#
+       :lines: 12-28
+       :emphasize-lines: 6, 15-16
+       :linenos:
+
+.. caution::
+    If there are multiple navigation properties defined between two types (i.e. more than one distinct pair of navigations that point to each other), then no relationships will be created by convention and you will need to manually configure them to identify how the navigation properties pair up.
+
+No Foreign Key Property
+^^^^^^^^^^^^^^^^^^^^^^^
+
+While it is recommended to have a foreign key property defined in the dependent entity class, it is not required. If no foreign key property is found, a shadow foreign key property will be introduced with the name ``<principal type name><principal key property name>`` (see :doc:`shadow-properties` for more information).
+
```

Comment:
Name would be `<Navigation property name><principal key property name>` if navigation pointing to principal is available.